content: Create FW Epilogue mission for Barmy Edwards (#5478)
endless-sky/endless-sky: data/human/weapons.txt

		"hull damage" 25
		"heat damage" 340
	description "The Plasma Turret is a pair of plasma cannons mounted on a turret so that they can fire in any direction. It is a fearsome weapon, but only the largest of ships have the space and the energy that it requires."
	
outfit "Plasma Repeater"
	category "Guns"
	cost 274000
	thumbnail "outfit/plasma_repeater"
	mass 32
	"outfit space" -32
	"weapon capacity" -32
	"gun ports" -1
	weapon
		sprite "projectile/plasma"
			"frame rate" 3
		sound "plasma"
		"hit effect" "plasma explosion"
		lifetime 38
		velocity 12
		inaccuracy 5
		reload 7
		"firing energy" 13
		"firing heat" 110
		"shield damage" 33
		"hull damage" 13
		"heat damage" 144
		"hit force" 19

> 	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the southern rim. The Plasma Repeater was the golden egg of Kraz Cybernetics and its ticket for interstellar government funding.`

Reviewer: ```suggestion
	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the Southern Rim. The Plasma Repeater was the golden egg of Kraz Cybernetics and its ticket for interstellar government funding.`
```
Is this supposed to be capitalized? Not sure, but something to check on if you haven't already.

> 	description `Derived from the original Plasma Cannon, this larger version trades off its larger bursts for a much faster rate of fire of less concentrated bursts. Although the smaller bursts of superheated plasma deal far less damage than its predecessor, it imparts more damage onto the enemy overall.`

Reviewer: ```suggestion
	description `Derived from the original Plasma Cannon, this larger version trades off its larger bursts for a much faster rate of fire. Although the smaller bursts of superheated plasma deal far less damage than its predecessor, it imparts more damage onto the enemy overall.`
```
Triple `bursts`.

		"heat damage" 340
	description "The Plasma Turret is a pair of plasma cannons mounted on a turret so that they can fire in any direction. It is a fearsome weapon, but only the largest of ships have the space and the energy that it requires."

outfit "Plasma Repeater"
	category "Guns"
	cost 274000

> 	thumbnail "outfit/plasma_repeater"

Reviewer 2: No underscores in images.
```suggestion
	thumbnail "outfit/plasma repeater"
```

> 	mass 32

Reviewer 3: ```suggestion
	"mass" 32
```

	"outfit space" -32
	"weapon capacity" -32
	"gun ports" -1
	weapon
		sprite "projectile/plasma"
			"frame rate" 3
		sound "plasma"
		"hit effect" "plasma explosion"
		lifetime 38
		velocity 12
		inaccuracy 5
		reload 7
		"firing energy" 13
		"firing heat" 110
		"shield damage" 33
		"hull damage" 13
		"heat damage" 144
		"hit force" 19

> 	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the Southern Rim. The Plasma Repeater was the golden egg of Kraz Cybernetics and its ticket for interstellar government funding.`

Reviewer 2: ```suggestion
	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the Southern Rim. The repeater proved immensely popular amongst starship captains, and was Kraz Cybernetics' ticket for interstellar government funding.`
```

> 	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the Southern Rim. The repeater proved immensely popular amongst starship captains, and was Kraz Cybernetics' ticket for interstellar government funding.`

Author: I took a pass at partially rewriting the first description, not sure if it fits or not so I'll wait on your thoughts.

			"frame rate" 3
		sound "plasma"
		"hit effect" "plasma explosion"
		"lifetime" 38
		"velocity" 12
		"inaccuracy" 5
		"reload" 7
		"firing energy" 13
		"firing heat" 100
		"shield damage" 33
		"hull damage" 13
		"heat damage" 144
		"hit force" 19
	description `Decades after buying the rights for the Plasma Cannon, Kraz Cybernetics put this weapon on the shelves of every high-end outfitter of the Southern Rim. The repeater has so far proved immensely popular amongst any who braved testing it, and is shaping up to become an essential deterrent for anyone who flaunts their haul for pirate plunder.`

> 	description `	Derived from the original Plasma Cannon, this heavier version trades off its larger bursts for a much faster rate of fire. Although the smaller bursts of superheated plasma deal far less damage than its predecessor, the volume of fire results in more damage to the target overall`

Reviewer: ```suggestion
	description `	Derived from the original Plasma Cannon, this heavier version trades off its larger bursts for a much faster rate of fire. Although the smaller bursts of superheated plasma deal far less damage than its predecessor, the volume of fire results in more damage to the target overall.`
```
